### PR TITLE
Chore/fix google auth

### PR DESCRIPTION
**Pull request by zalapeach**
## This pull request involves

* Change login for google_oauth_2
* Fix specs
* Adds new variables for login

## Description

Update google login auth for use google_oauth_2, for all the environments

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -10,7 +10,7 @@ gem 'bootstrap-sass'
 gem 'carrierwave'
 gem 'cocoon'
 gem 'coffee-rails', '~> 4.0.0'
-gem 'crowdint_auth', '0.1.3'
+gem 'crowdint_auth', '0.1.3', github: 'jmoralesc/crowdint_auth_new'
```
Comment (softr8) [inline Gemfile]: can we use crowdint organization ?

Comment (vicmaster) [inline Gemfile]: Agree with @softr8... @jmoralesc do you know if your PR were merged into crowdint_auth_new from original repo?

Comment (zalapeach) [inline Gemfile]: It is not merged yet... the tests are failing

Comment (vicmaster) [inline Gemfile]: kk and you need this to fix the @rodacato PR, right?

Comment (zalapeach) [inline Gemfile]: Correct

**Comment by vicmaster** [comment]
LGTM

**Pull request merged**